auraed/src/lib.rs

```diff
@@ -278,11 +295,20 @@ impl AuraedRuntime {
 
         let discovery_service = DiscoveryService::new();
         let discovery_service_server =
-            DiscoveryServiceServer::new(discovery_service.clone());
+            DiscoveryServiceServer::new(discovery_service);
         health_reporter
             .set_serving::<DiscoveryServiceServer<DiscoveryService>>()
             .await;
 
+        let observe_service = ObserveService::new(
+            Arc::new(LogChannel::new(String::from("TODO"))),
```

Comment:
Is the `Arc` needed? `LogChannel` can be cloned.

Reply:
I don't think that `LogChannel` is actually used. I just did the minimum to make it compile 😅